cmd/coordinator.go

```diff
@@ -12,13 +12,14 @@ import (
 )
 
 var (
-	walletFlagName       = "wallet"
-	hideSweptFlagName    = "hide-swept"
-	sortByAmountFlagName = "sort-amount"
-	headFlagName         = "head"
-	tailFlagName         = "tail"
-	feeFlagName          = "fee"
-	dryRunFlagName       = "dry-run"
+	walletFlagName        = "wallet"
+	hideSweptFlagName     = "hide-swept"
+	sortByAmountFlagName  = "sort-amount"
+	headFlagName          = "head"
+	tailFlagName          = "tail"
+	feeFlagName           = "fee"
+	dryRunFlagName        = "dry-run"
+	depositsCountFlagName = "deposits-count"
```

Comment:
What do you think about annotating the flags to which command they belong? It's getting quite crowded here.

```
var (
	// listDepositsCommand:
	// proposeDepositsSweepCommand:
	walletFlagName = "wallet"

	// listDepositsCommand:
	hideSweptFlagName    = "hide-swept"
	sortByAmountFlagName = "sort-amount"
	headFlagName         = "head"
	tailFlagName         = "tail"

	// proposeDepositsSweepCommand:
	feeFlagName    = "fee"
	dryRunFlagName = "dry-run"

	// estimateDepositsSweepFeeCommand:
	depositsCountFlagName = "deposits-count"
)
```

Non-blocking of course.

Reply:
See 49176d14